src/components/Header/index.css

```diff
@@ -0,0 +1,34 @@
+*{
+    box-sizing: border-box;
+}
+.header-section{
+    width: 100vw;
+    display: flex;
+    flex-wrap: wrap;
+    padding-top:10px;
+    align-items: center;
+    justify-content: space-between;
+    padding-left: 10px;
+    background-color: #16003B;
+}
```

Comment:
You can use variable names for colors, which makes it easy to edit and find which color is used where.